app/helpers/publish_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module PublishHelper
+  def old_publish_link_for(path)
+    "#{Settings.publish_url}#{path.gsub('publish/', '')}"
```

Comment:
```suggestion
    "#{Settings.publish_url}#{path.sub(/^publish\//, '')}"
```

replace first and begins with rather then end and all

Reply:
Just had to find something didn't you